File: 프로그래머스/lv0/120836. 순서쌍의 개수/순서쌍의 개수.js
function solution(n) {
    const sqrt = Math.sqrt(n);
    let result = 0;
    for(let i = 1; i <= sqrt; i++){
        if(n % i === 0) result++
    }
    return sqrt === Math.floor(sqrt) ? result * 2 - 1 : result * 2
}

File: 프로그래머스/lv0/120836. 순서쌍의 개수/순서쌍의 개수.py
def solution(n):
    sqrt = int(n ** 0.5)
    result = len([i for i in range(1, sqrt + 1) if n % i == 0]) * 2
    return  result - 1 if sqrt == n ** 0.5 else result

    #return len(list(filter(lambda v: n % (v+1) == 0, range(n))))